src/components/Dialog/Dialog.scss

```diff
@@ -148,7 +149,7 @@
 
   // Spacing between bottom buttons
   .ms-Dialog-action + .ms-Dialog-action {
-    margin: 0 0 0 16px;
+    @include margin-left(6px);
```

Comment:
One minor suggestion; these styles were ported from original fabric and haven't been completely scrubbed. We usually try to avoid asymmetrical paddings and margins. Would it be possible to achieve the same effect with symmetrical margins? E.g. if there is supposed to be margin of 6px between buttons, use a 3px margin on left/right and offset the container by -3px margin on left/right.

Avoiding asymmetry where possible helps, sometimes the RTL rules can cause very subtle difficulties later on in overriding values. Clearly not always avoidable.

If it's not possible, I'll take your word for it. :)